v0/destinations/marketo/transform.js

```diff
@@ -328,16 +350,42 @@ const processIdentify = async (message, formattedDestination, token) => {
   if (isDefinedAndNotNull(userId)) {
     inputObj.userId = userId;
   }
+  let endPoint = `https://${accountId}.mktorest.com/rest/v1/leads.json`;
+  let payload = {
+    action: "createOrUpdate",
+    input: [inputObj],
+    lookupField: "id"
+  };
+  // handled if vdm enabled
+  if (get(message, MappedToDestinationKey)) {
```

Comment:
This makes it a RETL feature only? Shouldn't we make it generic?

Reply:
For making this generic a plan should be there. changes in destination config and as a whole will be there. Changes are now made only for vdm support

Reply:
Ok We will take care of that makes sense